hasura.planx.uk/migrations/1731498447017_add template from column to flows/up.sql

```diff
@@ -0,0 +1 @@
+alter table "public"."flows" add column "templated_from" timestamp;
```

Comment:
Sorry think this might have been my typo on Trello but `templated_from` should actually be `uuid` type, not timestamp, same as `copied_from` !